README.md

```diff
@@ -1,36 +1,42 @@
-# Check Digit < Insert name > Library
+# Check Digit Spectral Config
 
-The Check Digit < Insert Name >  library is a library for Check Digit services to deal with [ short description about what the library does]
-  
-  Features:
-* Feature 1
-* Feature 2
-* Feature3
-* (...)
+Check Digit Spectral Config is the standard package to install and configure [Spectral](https://stoplight.io/open-source/spectral/) for use in Check Digit projects.
 
+### Installing
 
+Spectral Config should be installed as a dev dependency:
+`npm install @checkdigit/spectral-config --save-dev`
 
-### Installing
+### Additional Setup
 
-`npm install @checkdigit/< Insert name >` 
+Once installed, add a file named `.spectral.json` to the root of your project. This file should extend Spectral Config:
 
-### Use
-It could be. i.e.
+```jsonc
+{
+  "extends": ["@checkdigit/spectral-config"]
+}
 ```
-import * as insertName from '@checkdigit/< Insert name >';
 
-const someVariable = insertName();
+Also add the following to your `package.json` to run Spectral:
 
+```jsonc
+"scripts": {
+  // ...
+  "lint:spec": "spectral lint src/**/swagger.yml",
```

Comment:
should this be `lint:swagger` or `lint:spectral` or `lint:openapi`? `lint:spec` implies linting spec files.

Reply:
yep, that makes sense. changing to lint:openapi.